src/components/Page/components/Header/Header.scss

```diff
@@ -104,7 +104,7 @@ $action-menu-rollup-computed-width: icon-size();
   .mobileView & {
     position: absolute;
     top: spacing(loose) + (control-height() / 4);
-    right: 0;
+    right: spacing(loose);
```

Comment:
The padding is 20px around the header, this makes the rollup action consistent. 

The icon has some padding around it so it does not exactly line up with the edge of the padding. Maybe this would look better with a smaller padding so the icon lines up with the texts spacing?

Reply:
It does feel like _slightly_ too much spacing